apis/python/src/tiledbsoma/_index_util.py

```diff
@@ -26,3 +26,16 @@ def build_index(
     reindexer = clib.IntIndexer()
     reindexer.map_locations(keys, thread_count)
     return reindexer
+
+
+def build_index(
```

Comment:
Given that this function is only present in the tiledb variant of SOMA, we should follow the naming convention used for other tiledbsoma-exlusive functions.  Propose `tiledbsoma_build_index()`